contracts/protocol/facets/SellerHandlerFacet.sol

```diff
@@ -209,110 +205,134 @@ contract SellerHandlerFacet is SellerBase {
 
-                // Update admin
-                seller.admin = sender;
+                    preUpdateSellerCheck(_sellerId, msgSender(), lookups);
 
-                // Store new seller id by admin mapping
-                lookups.sellerIdByAdmin[sender] = _sellerId;
+                    // Delete old seller id by admin mapping
+                    delete lookups.sellerIdByAdmin[seller.admin];
 
-                // Delete pending update admin
-                delete sellerPendingUpdate.admin;
-                // Delete auth token for seller id if it exists
-                delete protocolEntities().authTokens[_sellerId];
-            }
+                    // Update admin
+                    seller.admin = msgSender();
 
-            // Approve operator update
-            if (role == SellerUpdateFields.Operator && sellerPendingUpdate.operator != address(0)) {
-                require(sellerPendingUpdate.operator == sender, UNAUTHORIZED_CALLER_UPDATE);
+                    // Store new seller id by admin mapping
+                    lookups.sellerIdByAdmin[msgSender()] = _sellerId;
 
-                preUpdateSellerCheck(_sellerId, sender, lookups);
+                    // Delete pending update admin
+                    delete sellerPendingUpdate.admin;
+                    // Delete auth token for seller id if it exists
+                    delete protocolEntities().authTokens[_sellerId];
 
-                // Delete old seller id by operator mapping
-                delete lookups.sellerIdByOperator[seller.operator];
+                    updateApplied = true;
+                }
 
-                // Update operator
-                seller.operator = sender;
+                // Approve operator update
+                if (role == SellerUpdateFields.Operator && sellerPendingUpdate.operator != address(0)) {
+                    require(sellerPendingUpdate.operator == msgSender(), UNAUTHORIZED_CALLER_UPDATE);
 
-                // Transfer ownership of NFT voucher to new operator
-                IBosonVoucher(lookups.cloneAddress[_sellerId]).transferOwnership(sender);
+                    preUpdateSellerCheck(_sellerId, msgSender(), lookups);
 
-                // Store new seller id by operator mapping
-                lookups.sellerIdByOperator[sender] = _sellerId;
+                    // Delete old seller id by operator mapping
+                    delete lookups.sellerIdByOperator[seller.operator];
 
-                // Delete pending update operator
-                delete sellerPendingUpdate.operator;
-            }
+                    // Update operator
+                    seller.operator = msgSender();
 
-            // Aprove clerk update
-            if (role == SellerUpdateFields.Clerk && sellerPendingUpdate.clerk != address(0)) {
-                require(sellerPendingUpdate.clerk == sender, UNAUTHORIZED_CALLER_UPDATE);
+                    // Transfer ownership of NFT voucher to new operator
```

Comment:
`// Transfer ownership of NFT voucher contract to new operator`

Reply:
done